bitmap/bitmap.go

```diff
@@ -0,0 +1,105 @@
+package bitmap
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/RoaringBitmap/roaring"
+)
+
+type Bitmap struct {
+	base  int32
+	size  int32
+	start int32
+	data  *roaring.Bitmap
+	lock  *sync.Mutex
+}
+
+// NewBitmap allocate a bitmap range from [start, end], notice the end is included
+func NewBitmap(start, end int32) *Bitmap {
+	size := end - start + 1
```

Comment:
How about adding input checks for `start` and `end`?

Reply:
And do what?  Log it?  Return an error?  I was careful to keep the external interface the same as the original.  Callers such as instance-manager and backing image manager call NewBitmap in their own constructors, having first done the necessary validation with a method such as `ParsePortRange`.

Since this is a common library, NewBitmap can return (*Bitmap, error) and I can then rearrange the places it is called.  WDYT?

Reply:
Ideally, the function should return an error if the parameters are invalid. However, since go-common-libs will be used by Longhorn components across different versions, adding an `error` in returned values breaks compatibility. Therefore, the only option is to return nil and log the error. WDYT?

Reply:
Done.